### PR TITLE
Add JsonAdaptedMeeting tests

### LINKED ISSUE #141: fix jsonStorage tests
right now there is no unit test coverage for the jsonadaptedpersons to include the jsonadapted meetings and there is completely no test cases for jsonadaptedmeetings, do come out with your own sample jsons to ensure that these edge cases pass and are caught

### PR DESCRIPTION
**Pull request by timothysashimi**
Fixes #141

### CODE REVIEW

File: src/test/java/seedu/address/storage/JsonAdaptedMeetingTest.java [hunk 1]
```diff
@@ -1,21 +1,19 @@
...
 
 import seedu.address.commons.exceptions.IllegalValueException;
-import seedu.address.model.person.Address;
-import seedu.address.model.person.Email;
-import seedu.address.model.person.Name;
-import seedu.address.model.person.Phone;
+import seedu.address.testutil.MeetingBuilder;
 
 public class JsonAdaptedMeetingTest {
     private static final String INVALID_NAME = "R@chel";
```
Comment (itstrueitstrueitsrealitsreal) [inline src/test/java/seedu/address/storage/JsonAdaptedMeetingTest.java]: Maybe this string could be removed if not in use?

File: src/test/java/seedu/address/storage/JsonAdaptedMeetingTest.java [hunk 2]
```diff
@@ -1,21 +1,19 @@
 package seedu.address.storage;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static seedu.address.storage.JsonAdaptedPerson.MISSING_FIELD_MESSAGE_FORMAT;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_DATETIME;
 import static seedu.address.testutil.Assert.assertThrows;
 import static seedu.address.testutil.TypicalPersons.JAMAL;
```
Comment (itstrueitstrueitsrealitsreal) [inline src/test/java/seedu/address/storage/JsonAdaptedMeetingTest.java]: Is this import being used?

File: src/test/java/seedu/address/storage/JsonAdaptedMeetingTest.java [hunk 3]
```diff
@@ -36,85 +34,49 @@ public class JsonAdaptedMeetingTest {
...
-        String expectedMessage = Email.MESSAGE_CONSTRAINTS;
-        assertThrows(IllegalValueException.class, expectedMessage, person::toModelType);
-    }
-
-    @Test
-    public void toModelType_nullEmail_throwsIllegalValueException() {
-        JsonAdaptedPerson person = new JsonAdaptedPerson(VALID_NAME, VALID_PHONE, null,
-                VALID_ADDRESS, VALID_TAGS, VALID_MEETINGS);
-        String expectedMessage = String.format(MISSING_FIELD_MESSAGE_FORMAT, Email.class.getSimpleName());
-        assertThrows(IllegalValueException.class, expectedMessage, person::toModelType);
```
Comment (itstrueitstrueitsrealitsreal) [inline src/test/java/seedu/address/storage/JsonAdaptedMeetingTest.java:91]: Good that the tests for the fields were updated!

**Review by itstrueitstrueitsrealitsreal — COMMENTED** [review]
LGTM other than some issues regarding unused imports and variables!

**Review by sethteo — COMMENTED** [review]
LGTM besides the issues Kenneth raised!

File: src/test/java/seedu/address/storage/JsonAdaptedMeetingTest.java (hunk 1 above)
Comment (timothysashimi) [reply src/test/java/seedu/address/storage/JsonAdaptedMeetingTest.java]: yup removed!
